pkgs/development/python-modules/flufl/lock.nix

```diff
@@ -1,14 +1,20 @@
-{ lib, buildPythonPackage, fetchPypi, pytestCheckHook
-, atpublic, psutil, pytest-cov, sybil
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, atpublic
+, psutil
+, pytest-cov
```

Comment:
```suggestion
```
Please patch out the usage of cov. We can't send it anywhere or do anything with it.